nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -92,13 +92,14 @@ fi
 mkdir -m 0755 -p $mountPoint/dev $mountPoint/proc $mountPoint/sys $mountPoint/etc $mountPoint/run $mountPoint/home
 mkdir -m 01777 -p $mountPoint/tmp
 mkdir -m 0755 -p $mountPoint/tmp/root
-mkdir -m 0755 -p $mountPoint/var
+mkdir -m 0755 -p $mountPoint/var/permissions-wrappers
 mkdir -m 0700 -p $mountPoint/root
 mount --rbind /dev $mountPoint/dev
 mount --rbind /proc $mountPoint/proc
 mount --rbind /sys $mountPoint/sys
 mount --rbind / $mountPoint/tmp/root
 mount -t tmpfs -o "mode=0755" none $mountPoint/run
+mount -t tmpfs -o "mode=0755" none $mountPoint/var/permissions-wrappers
```

Comment:
Good question, we might not? I preserved it since it was in the original and I remember the `/var/setuid-wrappers` directory being mounted in the nixos-install.sh script.

I'll go look at my other code to see if it's needed. A lot of things have changed rather rapidly, thanks for pointing this out.